envoy/upstream/host_description.h

```diff
@@ -119,6 +120,11 @@ class HostDescription {
    */
   virtual Network::Address::InstanceConstSharedPtr address() const PURE;
 
+  /**
+   * @return the address used to connect to the host.
```

Comment:
Different comment? Maybe a bit about how this is intended to be used?

Reply:
Done.